### PR TITLE
feat : added typography

### PR DESCRIPTION
**Pull request by sach10-create**

### CODE REVIEW

**Comment by saurabhan** [comment]
Typography button links to snackbar page.

**Comment by Ranjeetk489** [comment]
Your title on landing page is document

File: components/typography/typography.html [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nova UI</title>
+    <link rel="icon"
+        href="../assets/chip-img.png"
+        type="image/x-icon">
+    <!-- <link
+    rel="stylesheet"
```
Comment (Ranjeetk489) [inline components/typography/typography.html:13]: remove comment

**Review by Ranjeetk489 — APPROVED** [review]
After fixing the wrong link redirection the code will be good to merge

**Pull request merged**